e2e/claims.service.e2e.ts

```diff
@@ -825,109 +825,103 @@ describe('Сlaim tests', () => {
+        await signerService.connect(staticIssuer, ProviderType.PrivateKey);
+        await createExampleSignedCredential([], roleName);
+        const { issuedToken } = await enrolAndIssue(rootOwner, staticIssuer, {
+          subjectDID: rootOwnerDID,
+          claimType: roleName,
+          registrationTypes: [
+            RegistrationTypes.OnChain,
+            RegistrationTypes.OffChain, // role type issuer should have offchain claim
+          ],
+          publishOnChain: true,
+          issuerFields: [],
+        });
+        await signerService.connect(rootOwner, ProviderType.PrivateKey);
+        let subjectDoc = await didRegistry.getDidDocument({
+          did: rootOwnerDID,
+          includeClaims: true,
+        });
+        mockCachedDocument.mockResolvedValueOnce(subjectDoc);
 
-      subjectDoc = await didRegistry.getDidDocument({
-        did: rootOwnerDID,
-        includeClaims: true,
-      });
-      const result = await claimsService.resolveCredentialAndVerify(
-        rootOwnerDID,
-        roleName
-      );
-      expect(result.errors).toHaveLength(0);
-      expect(result.isVerified).toBeTruthy;
-      expect(claimsService.verifyVc).toHaveBeenCalled;
-    });
-    test('resolveAndVerify should resolve an EIP191JWT and verify the claim', async () => {
-      const roleName = `${verifyOffChainClaimRole}.${root}`;
-      await domainsService.createRole({
-        roleName: verifyOffChainClaimRole,
-        namespace,
-        data: roles[roleName],
-        returnSteps: false,
-      });
+        await claimsService.publishPublicClaim({
+          claim: { token: issuedToken },
+        });
 
-      await signerService.connect(staticIssuer, ProviderType.PrivateKey);
-      const { issuedToken } = await enrolAndIssue(rootOwner, staticIssuer, {
-        subjectDID: rootOwnerDID,
-        claimType: roleName,
-        registrationTypes: [
-          RegistrationTypes.OnChain,
-          RegistrationTypes.OffChain,
-        ],
-        publishOnChain: true,
-        issuerFields: [],
-      });
-      await signerService.connect(rootOwner, ProviderType.PrivateKey);
-      let subjectDoc = await didRegistry.getDidDocument({
-        did: rootOwnerDID,
-        includeClaims: true,
-      });
-      mockCachedDocument.mockResolvedValueOnce(subjectDoc);
+        subjectDoc = await didRegistry.getDidDocument({
+          did: rootOwnerDID,
+          includeClaims: true,
+        });
+        const result = await claimsService.resolveCredentialAndVerify(
+          rootOwnerDID,
+          roleName
+        );
+        expect(result.errors).toHaveLength(0);
+        expect(result.isVerified).toBeTruthy;
+        expect(claimsService.verifyVc).toHaveBeenCalled;
+      });
+      test('resolveAndVerify should resolve an EIP191JWT and verify the claim', async () => {
+        const roleName = `${verifyOffChainClaimRole}.${root}`;
+        await signerService.connect(staticIssuer, ProviderType.PrivateKey);
```

Comment:
Are you connecting to static issuer to issue? Issuer is connected in `enrolAndIssue`